search-processors/src/main/java/org/opensearch/searchpipelines/questionanswering/generative/llm/DefaultLlmImpl.java

```diff
@@ -118,6 +118,17 @@ protected Map<String, String> getInputParameters(ChatCompletionInput chatComplet
                             chatCompletionInput.getContexts()
                         )
                 );
+        } else if (chatCompletionInput.getModelProvider() == ModelProvider.OCI_GENAI) {
```

Comment:
Can you provide some tutorial/example for OCI_GENAI ?

Reply:
Yeah I added